app/src/main/java/com/aerogear/androidshowcase/features/authentication/AuthenticationFragment.java

```diff
@@ -49,9 +48,6 @@
     @Inject
     Navigator navigator;
 
-    @Inject
-    AppConfiguration appConfiguration;
```

Comment:
so it's not used in this file? What's the exception that caused crash?

Reply:
When it tries to read the service config it returns null so null reference is thrown

Reply:
I don't think remove it here is the right fix.

If I am not mistaken, the problem is this line here: https://github.com/aerogear/android-showcase-template/blob/master/app/src/main/java/com/aerogear/androidshowcase/domain/configurations/AppConfiguration.java#L25.

The problem is that in that line, it tries to load configuration using the type value "notes-service", but it doesn't exist in this file: https://github.com/aerogear/android-showcase-template/blob/master/app/src/main/assets/mobile-services.json.

So the right fix is to update that file, and change the `id` value of this line (https://github.com/aerogear/android-showcase-template/blob/master/app/src/main/assets/mobile-services.json#L8) to `notes-service`, and update the code to use `getServiceConfigurationById`.

Reply:
Ya this was just a quick fix so I didn't change the config file for that is because I don't know anything about that service or how its used.  What should the `name` become then, just swap the `name` and `id` values?

Does the auth fragment use `custom-runtime-connector` service?

Reply:
the `name` field doesn't matter, you can keep as it is. I don't think the auth fragment is using the `custom-runtime-connector`.